Mod Formats/FOMod/FOMod.cs

```diff
@@ -752,40 +636,36 @@ protected void LoadInfo()
 		{
 			if (ContainsFile("fomod/info.xml"))
 			{
-				XmlDocument xmlInfo = new XmlDocument();
-				xmlInfo.LoadXml(TextUtil.ByteToString(GetFile("fomod/info.xml")));
-				LoadInfo(xmlInfo, false);
+                try
+                {
+                    var xmlInfo = new XmlDocument();
+                    xmlInfo.LoadXml(TextUtil.ByteToString(GetFile("fomod/info.xml")));
+                    LoadInfo(xmlInfo, false);
+                }
+                catch (XmlException e)
```

Comment:
Here's the interesting code change, catching and wrapping the XML parsing error to give more info to the user, and log some info for troubleshooting.